apps/meteor/server/features/EmailInbox/EmailInbox_Incoming.ts

```diff
@@ -211,7 +217,7 @@ export async function onEmailReceived(email: ParsedMail, inbox: string, departme
 			],
 			rid,
 			email: {
-				references,
+				thread,
```

Comment:
Changing this prop works, but will make all the current email inboxes to stop "working", cause the thread won't be there (will be in references)

Reply:
as per our talk, i made the changes to allow for the old reference prop.